docs/sources/write/tooling-and-workflows/_index.md

```diff
@@ -143,6 +143,39 @@ Run `git fetch` before `git status` for the most accurate status.
 
 To understand the output of `git status`, refer to [Git - git-status Documentation](https://git-scm.com/docs/git-status#_output).
 
+### Checkout a PR branch from a fork
+
+> A fork is a new repository that shares code and visibility settings with the original "upstream" repository.
+
+For more information, refer to [About forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks).
+
+To checkout a PR branch from a fork, use the GitHub CLI (`gh`) tool.
+To install the GitHub CLI tool, refer to [Installation](https://github.com/cli/cli#installation).
+
+Replace _`PR NUMBER`_ with the number of the pull request.
+Run the command from a directory within the repository.
```

Comment:
Are we missing a step here where you have to copy the forked repo to your local?  Otherwise I don't see how you could run the command from within the repository...

Reply:
The `gh` CLI does all the fancy git stuff for you which is really handy. It fetches from the fork remote repository and configures a local branch in your repository to track that remote branch in the fork.

Reply:
Ah, then maybe mention that in an overview sentence before the step to download the tool?  (As previously mentioned, I haven't ever used it and don't know anything about the features or capabilities).

Reply:
Great idea, added in [ea68476](https://github.com/grafana/writers-toolkit/pull/365/commits/ea6847665a564c2305c8563e7955f553b40982b4)